client/app/telecom/telephony/line/management/mgcpIpRestriction/telecom-telephony-line-mgcpIpRestriction.js

```diff
@@ -0,0 +1,14 @@
+angular.module("managerApp").config(function ($stateProvider) {
+    "use strict";
```

Comment:
You can use an arrow function `() => { ... }` and remove "use strict", it's implicit :)

Reply:
As it's a new file, there is no mix of ES5 and ES6 ;-) But I prefer to keep all the section with ES5 for the moment. Migration will be done in the future :-)